src/test/java/com/tom/driver/manager/DriverUtils.java

```diff
@@ -0,0 +1,11 @@
+package com.tom.driver.manager;
+
+public class DriverUtils {
+    public static void setInitialConfiguration(){
```

Comment:
This one can be moved to DriverManager class so that getWebDriver will return already configured WebDriver instance